wwwapp/views.py

```diff
@@ -264,6 +264,8 @@ def can_edit_workshop(workshop, user):
 @login_required()
 def workshop_participants_view(request, name):
     workshop = get_object_or_404(Workshop, name=name)
+    if not workshop.is_publicly_visible():  # Zaakceptowane lub odwołane
```

Comment:
Use english in comment

Reply:
Go and fix all occurrences of this exact comment, I literally copy-pasted these lines from elsewhere without changes